environment.yml

```diff
@@ -1,10 +1,14 @@
 name: pancancer-classifier
+channels:
+- r
+- bioconda
 dependencies:
+- r-checkpoint=0.4.0
```

Comment:
I now specify channels for each package in environment. For example:

```yml
- r::r-checkpoint=0.4.0
- anaconda::jupyter=1.0.0
```

I find it makes it much easier to see where these things are coming from.